src/helperFunctions/types.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+from multiprocessing.sharedctypes import Synchronized, SynchronizedArray
+from multiprocessing.synchronize import Event
+from tempfile import _TemporaryFileWrapper
+from typing import Dict, List, Protocol, TypeVar, NamedTuple, TYPE_CHECKING, Union
+from unittest.mock import _patch
+
+
+if TYPE_CHECKING:
+    from analysis.plugin import AnalysisPluginV0
+
+KT = TypeVar('KT', str, tuple, bytes)  # generic key type
+VT = TypeVar('VT')  # generic value type
```

Comment:
Isn't it better to be explicit about the types?

Reply:
This is for the generic case (i.e. the function accepts all dicts but you want to annotate e.g. that it will return the same type as the values of the dict) https://docs.python.org/3/library/typing.html#generics